crates/dojo-world/src/manifest.rs

```diff
@@ -31,7 +32,7 @@ pub const EXECUTOR_CONTRACT_NAME: &str = "dojo::executor::executor";
 pub const BASE_CONTRACT_NAME: &str = "dojo::base::base";
 
 #[derive(Error, Debug)]
-pub enum ManifestError {
+pub enum AbstractManifestError {
```

Comment:
With your rework, we may keep `ManifestError` WDYT?

Reply:
i think not because there is still a `Manifest` struct and this particular error type can be returned while trying to load a remote world (from `RemoteLoadable` trait) so its not specifically related to `Manifest` struct.